README.md

```diff
@@ -1,4 +1,23 @@
 # Backend app
 
-This app was created with `@command-shift/create-backend-app`
-# music-library
+Project title: Music Library
```

Comment:
This really should be updated as there are some important bits of info missing. Some things that would make it much better:
- A description of what the app actually does and what its features are
- installation instructions (especially regarding the database and the need to create environment variables),
- what technologies are required to use it, what the application dependencies are (`Express` and `MySQL2`), and what the development dependencies are (`nodemon`, `dotenv`, `mocha`, `chai`, `superTest`)
- information about the routes provided by the app, what type of HTTP verb they all require and an example request body would be a great addition,
- how to test the app,
- what you would do if you had more time (if applicable)

Also, it would be worth looking into using markdown language for this as thats one of the most popular and widely used ways to construct your README. Have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) for some details on the techniques, also [this past students readme](https://github.com/cocomarine/music-library) for an example of what you can do with this